src/gui/Gui.cpp

```diff
@@ -3,84 +3,313 @@
 /// Copyright (c) 2021-present Anonymous275 read the LICENSE file for more info.
 ///
 
-#define _CRT_SECURE_NO_WARNINGS
 #include <wx/wxprec.h>
-#include <set>
 #ifndef WX_PRECOMP
-#include <wx/animate.h>
-#include <wx/mstream.h>
+#include <wx/dc.h>
+#include <wx/dcbuffer.h>
+#include <wx/fs_inet.h>
+#include <wx/graphics.h>
+#include <wx/hyperlink.h>
+#include <wx/sizer.h>
+#include <wx/statline.h>
+#include <wx/webview.h>
 #include <wx/wx.h>
-#include "gifs.h"
+#include <wx/wxhtml.h.>
 
+#include "Launcher.h"
+#include "Logger.h"
 #endif
```

Comment:
It looks like Launcher and Logger are not included if WX_PRECOMP is defined, see like 7 for the start of this ifndef

Reply:
Yes no worries, these headers are not yet used because the launch button code is just commented out for now due to its improper function. When the launch button is properly working, these headers might even change but will be fixed. Thank you!